data/templates/ants/config.yml

```diff
@@ -0,0 +1,10 @@
+name: What is this, a Center for Ants?!
+default:
+- WHAT IS THIS
+- A CENTER FOR ANTS
+link: http://i1.kym-cdn.com/entries/icons/original/000/013/833/c.jpg
```

Comment:
This link should do a bit more to explain the meme. Perhaps use http://knowyourmeme.com/memes/what-is-this-a-center-for-ants?